### PR TITLE
BigAnimal: CLI updates for 3.0

### PR DESCRIPTION
**Pull request by francoughlin**
## What Changed?

CLI updates for 3.0 - https://enterprisedb.atlassian.net/browse/UPM-23240

### CODE REVIEW

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 1]
```diff
@@ -4,7 +4,9 @@ navTitle: Managing clusters
 deepToC: true
 ---
 
-These examples show Azure as the cloud provider unless indicated otherwise. The functionality is the same when using AWS or Google Cloud.
+These examples show Azure as the cloud provider unless indicated otherwise. 
+
+Although the functionality is the same when using AWS or Google Cloud, there may be additional input flags based on cloud provider type. Use the `-h` or `--help` flags for more information on the CLI commands.
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:9]:
```suggestion
Although the functionality is the same when using AWS or Google Cloud, there may be additional input flags based on the cloud provider type. Use the `-h` or `--help` flags for more information on the CLI commands.  
```

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 2]
```diff
@@ -126,15 +128,15 @@ To enable you to view valid values to use in the configuration file for BigAnima
...
-│ ha     │ High Availability         │ enabled  │
-│ pgd    │ Extreme High Availability │ disabled │
-│ single │ Single Node               │ enabled  │
-└────────┴───────────────────────────┴──────────┘
+┌───────────────────────────────────────────────────────────────┐
+│ Architecture                                                  │
+├────────┬───────────────────────────────────────────┬──────────┤
+│ ID     │ Name                                      │ Status   │
+├────────┼───────────────────────────────────────────┼──────────┤
+│ ha     │ Primary/Standby High Availability         │ enabled  │
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:136]: I don't believe the name change was implemented in 3.0. See https://enterprisedb.atlassian.net/browse/UPM-23196.

**Review by drothery-edb — APPROVED** [review]

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx (hunk 2 above)
Comment (francoughlin) [reply product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:136]: Ah, I see. I did not know that. Thanks! Will fix.

**Review by edbamolvyavahare — CHANGES_REQUESTED** [review]
- needs update for cluster show-monitoring-urls output
- need to update all commands for new fields i.e  csp-subscription-ids and service-account-ids as mentioned [here](https://enterprisedb.atlassian.net/browse/UPM-23240?focusedCommentId=472725)
- Rename High Availability -> Primary/Standby High Availability at all places

**Comment by nidhibhammar** [comment]
@edbamolvyavahare, Please review the updated content and let me know if any further changes are required or are we good to publish this content?

File: product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx [hunk 3]
```diff
@@ -23,10 +25,16 @@ For example, to create a primary/standby high-availability cluster:
...
+Cluster architecture: Primary/Standby High Availability
 Number of standby replicas: 2 Replicas
 Enable read-only workloads: No
 Provider: Azure
+Cloud provider subscription ID:
+- 123123123
+- 456456456
+Service account IDs:
+- service-account-1234567b@development-data-123456.iam.gserviceaccount.com
+- service-account-1234567b@development-data-123456.iam.gserviceaccount.com
```
Comment (edbamolvyavahare) [inline product_docs/docs/biganimal/release/reference/cli/managing_clusters.mdx:37]: - input displayed from user is incorrect
- interactive mode only numbers separated by comma is a valid input 
- service a/c id prompt is incorrect

*kindly refer the confluence page spec diff or maybe please run the CLI binary in the interactive mode to understand the context

File: product_docs/docs/biganimal/release/reference/cli/using_features.mdx [hunk 4]
```diff
@@ -24,6 +24,12 @@ __OUTPUT__
 ? Volume type: Azure Premium Storage
 ? Volume properties: P1 (4 Gi, 120 Provisioned IOPS, 25 Provisioned MB/s)
 ? Networking: Public
+? Cloud Provider Subscription ID: 
+- 123123123
+- 456456456
+? Service Account IDs, (leave empty to stop adding):
+- service-account-1234567b@development-data-123456.iam.gserviceaccount.com
+- service-account-1234567b@development-data-123456.iam.gserviceaccount.com
```
Comment (edbamolvyavahare) [inline product_docs/docs/biganimal/release/reference/cli/using_features.mdx:32]: [same as above](https://github.com/EnterpriseDB/docs/pull/4812/files#r1378600370)

File: product_docs/docs/biganimal/release/reference/cli/using_features.mdx [hunk 5]
```diff
@@ -70,6 +76,12 @@ __OUTPUT__
 ? Volume properties: P1 (4 Gi, 120 Provisioned IOPS, 25 Provisioned MB/s)
 ? Networking: Public
 ? By default your cluster allows all inbound communications, add IP allowed list to restrict the access: No
+? Cloud Provider Subscription ID: 
+- 123123123
+- 456456456
+? Service Account IDs, (leave empty to stop adding):
+- service-account-1234567b@development-data-123456.iam.gserviceaccount.com
+- service-account-1234567b@development-data-123456.iam.gserviceaccount.com
```
Comment (edbamolvyavahare) [inline product_docs/docs/biganimal/release/reference/cli/using_features.mdx:84]: [same as above](https://github.com/EnterpriseDB/docs/pull/4812/files#r1378600370)

**Comment by ebgitelman** [comment]
@edbamolvyavahare Is this okay now? Thanks!

File: product_docs/docs/biganimal/release/reference/cli/using_features.mdx (hunk 5 above)
Comment (ebgitelman) [reply product_docs/docs/biganimal/release/reference/cli/using_features.mdx]: Should be good now.

**Review by edbamolvyavahare — APPROVED** [review]
LGTM

**Review by nidhibhammar — APPROVED** [review]

**Pull request merged**